core/payment-driver/zksync/src/zksync/wallet.rs

```diff
@@ -333,6 +333,6 @@ pub async fn withdraw<S: EthereumSigner + Clone>(
         .await
         .map_err(GenericError::new)?;
 
-    debug!("Withdraw handle: {:?}", withdraw_handle);
+    // TODO: Do we need this handle debugged? debug!("Withdraw handle: {:?}", withdraw_handle);
```

Comment:
Not really.

Reply:
made it a cleaner message and kept it, please check